src/pitch.ts

```diff
@@ -591,7 +591,48 @@ export class Pitch extends prebase.ProtoM21Object {
     getLowerEnharmonic(inPlace=false) {
         return this._getEnharmonicHelper(inPlace, -1);
     }
-    /* TODO: isEnharmonic, getEnharmonic, getAllCommonEnharmonics */
+
+    getEnharmonic(inPlace=false): Pitch|undefined {
```

Comment:
One difference between Python and Javascript is that Javascript tends to emphasize allowing chaining more than Python.  Most Javascript methods that do things like `obj.setFileType('.jpg')` expect that `obj` will be returned afterwards so you can do `obj.setFileType('.jpg').setColorSpace('BW')...`.  Hence one difference between music21(p) and music21j is that even when inPlace=True, we return a Pitch.

The necessity of a semicolon to end a line (esp. in real Java) and the emphasis on getter and setter functions is one of the reasons for the change.

Reply:
Fixed.